nion/swift/DocumentController.py

```diff
@@ -888,9 +888,9 @@ def export_file(self, display_item: DisplayItem.DisplayItem) -> None:
 
     def export_files(self, display_items: typing.Sequence[DisplayItem.DisplayItem]) -> None:
         if len(display_items) > 1:
-            export_dialog = ExportDialog.ExportDialog(self.ui, self)
-            export_dialog.on_accept = functools.partial(export_dialog.do_export, display_items)
-            export_dialog.show()
+            export_dialog = ExportDialog.ExportDialog(self.ui, self, self, display_items)
+            # export_dialog.on_accept = functools.partial(export_dialog.do_export, display_items)
+            # export_dialog.show()
```

Comment:
As a general note, can you remove unused code unless there is a compelling reason to leave it around in comments?